CMakeLists.txt

```diff
@@ -325,8 +325,8 @@ if(ENABLE_DATA_TOOLS)
     set_target_properties(${program} PROPERTIES FOLDER "Data Tools")
     target_include_directories(${program} PUBLIC ${VALHALLA_SOURCE_DIR}/third_party/cxxopts/include)
     target_link_libraries(${program} valhalla)
-    if (LUAJIT_FOUND AND APPLE)
-      # Using LuaJIT on macOS requires a couple of extra linker flags
+    if (LUAJIT_FOUND AND APPLE AND CMAKE_HOST_SYSTEM_PROCESSOR EQUAL "x86_64")
```

Comment:
i guess it cant be this!

```
static:~ distiller$ uname -m
x86_64
```

Reply:
Most likely I should use STREQUAL, but not EQUAL here. I'll fix that a bit later.